### ISSUE TITLE
comma-spacing in template strings: false negative

### ISSUE THREAD

**Issue by kmoe** [eslint/eslint#4098]
**What version are you using?**
eslint v0.20.0 with babel-eslint (happy to raise an issue there if it is a bug with babel-eslint instead)

**What did you do?**

created the file template_comma.js:

```javascript
var foo = 'bar';
var template = `template string with ${foo}, followed by a comma`;
console.log(template);
```

with an .eslintrc of this:

```javascript
{
  "parser": "babel-eslint",

  "env": {
    "browser": true,
    "node": true,
    "es6": true
  },

  "rules": {
      "semi": [2, "always"],
      "quotes": [2, "single"]
  }
}
```

then ran this in the terminal:
```
eslint template_comma.js
```

**What happened?**

eslint stdout output:

```
template_comma.js
  2:43  error  A space is required after ','  comma-spacing

✖ 1 problem (1 error, 0 warnings)
```

**What did you expect to happen?**

no errors

**Analysis**

It looks like the comma-spacing rule doesn't account for legitimate uses of commas when variables are interpolated in template strings.

**Comment by gyandeeps** [comment]
Thats a long way back in term of releases. I would recommend upgrading eslint to `1.6` and then try this.

**Comment by hzoo** [comment]
I fixed this a while ago - same as https://github.com/babel/babel-eslint/issues/31? (try upgrading to latest of both eslint/babel-eslint)

**Comment by ilyavolodin** [comment]
It looks like this issue has been fixed. @kmoe feel free to reopen if you are still seeing it with newer versions for eslint and babel-eslint.

**Comment by kmoe** [comment]
Thanks all, didn't realise how far back I was in versions. Will upgrade.
